web/src/components/SubscriptionFormHTML.jsx

```diff
@@ -1,20 +1,30 @@
-import { Button, Input, Space } from "antd";
+import { Typography } from "antd";
+
+import { ThemedButton } from "./ThemedButton";
+import { ThemedInput } from "./ThemedInput";
+
+import "./SubscriptionFormHTML.css";
 
 const SubscriptionFormHTML = () => {
 	return (
-		<div>
+		<div className="subscription-form__container">
 			<form action="/api/subscribe" method="post" id="subscription-form">
-				<Space.Compact>
-					<Input
+				<div className="subscription-form__layout">
+					<ThemedInput
 						type="email"
 						name="email"
-						placeholder="Enter your email"
+						placeholder="Enter your email linked to CYF Slack"
 						required
 					/>
-					<Button type="primary" htmlType="submit">
-						Subscribe
-					</Button>
-				</Space.Compact>
+					<ThemedButton type="primary" htmlType="submit">
+						<Typography.Title
```

Comment:
Instead of wrapping text inside <Typography.Title> within the button, use<Typography.Text> for better alignment. what do you think?

Reply:
I used `Typography.Title` to match our Figma spec (H5/regular) and maintain semantic HTML structure, as `Typography.Text` doesn't provide h5 semantics. Could you elaborate on the alignment issues you're seeing? I'd like to understand the specific alignment problems to address them while still following our design requirements.